packages/zowe-explorer-api/package.json

```diff
@@ -16,7 +16,9 @@
     "@types/semver": "^7.5.0"
   },
   "dependencies": {
+    "@types/vscode": "^1.80.0",
```

Comment:
Dumb questions:
- will there be issues for developers who have `"@types/vscode": "<1.80.0"` ?
- what about their  `"vscode": ""<1.80.0"` ?

Reply:
definitely not dumb questions - didn't consider the version of `@types/vscode` that we have in the package root. I've updated the versions to match what's in our "lerna" `package.json` 👍